### ISSUE TITLE
unrecognized RUBY_VERSION 2.3.0

### ISSUE THREAD

**Issue by segantii-alex-falkowski** [DamirSvrtan/fasterer#34]
```
bundle exec fasterer

/home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/ruby_parser-3.7.2/lib/ruby_parser_extras.rb:1393:in `for_current_ruby': unrecognized RUBY_VERSION 2.3.0 (RuntimeError)
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/parser.rb:8:in `parse'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/analyzer.rb:21:in `scan'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/file_traverser.rb:58:in `scan_file'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/file_traverser.rb:50:in `block in traverse_files'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/file_traverser.rb:50:in `each'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/file_traverser.rb:50:in `traverse_files'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/file_traverser.rb:27:in `traverse'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/lib/fasterer/cli.rb:7:in `execute'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/gems/fasterer-0.3.1/bin/fasterer:5:in `<top (required)>'
	from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/bin/fasterer:23:in `load'

from /home/ubuntu/optimize_it/vendor/bundle/ruby/2.3.0/bin/fasterer:23:in `<main>' bundle exec fasterer returned exit code 1
```

**Comment by segantii-alex-falkowski** [comment]
Looks like this issue has been raised https://github.com/seattlerb/ruby_parser/issues/205

**Comment by ioquatix** [comment]
+1 :)

**Comment by DamirSvrtan** [comment]
Until the issue is solved internally in ruby_parser, I'm unable to do anything.

**Comment by ioquatix** [comment]
I feel your pain. Sorry.

**Comment by lienvdsteen** [comment]
+1

**Comment by rhymes** [comment]
@DamirSvrtan ruby_parser just released support for 2.3.0 in version 3.8.0 https://rubygems.org/gems/ruby_parser/versions/3.8.0

**Comment by DamirSvrtan** [comment]
@segantii-alex-falkowski @ioquatix @lienvdsteen @rhymes please install ruby_parser version 3.8.1 to work with Ruby 2.3.0. I've added the test and it works.

I don't want to put a more stricter dependency in fasterer with ruby_parser with '>= 3.8.1'. 

So if you're using it without bundler just do:

```shell
gem install ruby_parser -v 3.8.1
```

Or if you're using it in bundler,

```shell
bundle update ruby_parser
```
This is a problem with a dependency, not with fasterer, so i think i shouldn't do anything here.
